requirements/production.txt

```diff
@@ -5,3 +5,5 @@ gevent
 gunicorn
 mysqlclient
 PyYAML
+python-memcached
+sqlparse
```

Comment:
Any idea why `sqlparse` is only needed in production? It seems like it should be needed anytime we run migrations. Maybe one of the other development dependencies installs it? I'm asking because it seems like we should move it to `base.txt` instead.

Reply:
Yes the `django-debug-toolbar` package included in `local.txt` installs the `sqlparse` module

Reply:
Ok, but if we need it as well, it's better just to put it in `base.txt` rather than relying on django-debug-toolbar to install it sometimes.

Reply:
Done